test/app/views/layouts/preview.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>ViewComponent - Preview - Test</title>
+</head>
+
+<body>
+  <%= yield %>
+  <%= content_for?(:view_component_preview_source) ? yield(:view_component_preview_source) : yield %>
```

Comment:
Isn't this code double yielding if there is no `view_component_preview_source`

Reply:
Fixed in 6b9fccd62f888cca8bbcf3889e1f56734897cd37